lib/RV/Admin/DateTimeFormat.php

```diff
@@ -90,6 +90,66 @@ public static function formatUTCTime($sDateTime)
         return $oTime->format("$time_format");
     }
 
+    /**
+     * A method that takes a date/time string and formats it using the user's
+     * locale information - i.e. using the appropriate date/time representation.
+     *
+     * @param string dateTime The date/time to be formatted, in a string format
+     * @return string The date/time in the user's locale format
+     */
+    public static function formatDateTime($sDateTime)
+    {
+        // Convert the date/time string to a PEAR Date object
+        $oDateTime = new Date($sDateTime);
+        // Obtain the globally set $date_format and $time_format values
+        // from the user's locale which defines how the date/time output
+        // string should be formatted
+        global $date_format, $time_format;
+        // Return the converted PEAR Date object in the appropriate locale
+        // format for date/time values
+        return $oDateTime->format("$date_format $time_format");
```

Comment:
I know it's mostly copied from above, but possibly the new methods can be just a single self explanatory line:

`return (new Date($sDateTime))->format("{$GLOBALS['date_format']} {$GLOBALS['time_format']}");`

Reply:
Agreed; will also refactor to "private" methods, as the same code is needed regardless of if it is a UTC or local timezone date/time that needs formatting according to the user's locale settings. Will update.